From 925c1c836becbf2523d16a4944bfda60b308ce6c Mon Sep 17 00:00:00 2001
From: Uri Eliyahu <uriel@marvell.com>
Date: Thu, 8 Aug 2013 16:53:34 +0300
Subject: [PATCH 0937/1825] pp2: fix policer drop reg mask

https://github.com/MISL-EBU-System-SW/misl-windriver.git linux-3.4.69-14t2-read
commit 1e2cbd48e5250dddea14dd1bd340dee066d09e50

	fix policer drop threshold field size

Change-Id: I77efd8dacb02110450572e3bb2fb166d6ccf3810
Signed-off-by: Uri Eliyahu <uriel@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/2991
Reviewed-by: Igor Patrik <igorp@marvell.com>
Tested-by: Igor Patrik <igorp@marvell.com>
Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 arch/arm/plat-armada/mv_hal/pp2/plcr/mvPp2PlcrHw.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/plat-armada/mv_hal/pp2/plcr/mvPp2PlcrHw.h b/arch/arm/plat-armada/mv_hal/pp2/plcr/mvPp2PlcrHw.h
index 6bf6c2e..0d31a3a 100644
--- a/arch/arm/plat-armada/mv_hal/pp2/plcr/mvPp2PlcrHw.h
+++ b/arch/arm/plat-armada/mv_hal/pp2/plcr/mvPp2PlcrHw.h
@@ -148,7 +148,7 @@ extern "C" {
 #define MV_PP2_PLCR_EDROP_THRESH_NUM		4
 
 #define MV_PP2_PLCR_EDROP_TR_OFFS(i)		((i % 2) ? 16 : 0)
-#define MV_PP2_PLCR_EDROP_TR_BITS		16
+#define MV_PP2_PLCR_EDROP_TR_BITS		14
 #define MV_PP2_PLCR_EDROP_TR_ALL_MASK(i)	(((1 << MV_PP2_PLCR_EDROP_TR_BITS) - 1) << MV_PP2_PLCR_EDROP_TR_OFFS(i))
 #define MV_PP2_PLCR_EDROP_TR_MASK(i, tr)	(((tr) << MV_PP2_PLCR_EDROP_TR_OFFS(i)) & MV_PP2_PLCR_EDROP_TR_ALL_MASK(i))
 
-- 
1.7.5.4

